package.json

```diff
@@ -20,7 +20,7 @@
     "lint:js": "eslint test scripts webpack.config.js demos/webpack.config.js karma.conf.js",
     "lint:ts": "tslint --exclude \"**/*.d.ts\" \"packages/**/*.ts\" \"test/**/*.ts\" \"scripts/**/*.ts\"",
     "lint:html": "find test/screenshot/spec/ -name '*.html' | grep -v 'index.html$' | xargs htmllint",
-    "lint:imports": "node scripts/lint-imports.js",
+    "lint:mdc": "ts-node-dev --project scripts/tsconfig.json -- scripts/lint-mdc.ts",
```

Comment:
`ts-node-dev` is no longer necessary thanks to https://github.com/material-components/material-components-web/pull/4481#discussion_r263476214.

Replaced with `ts-node`.

I originally chose `ts-node-dev` because [`ts-node`](https://github.com/TypeStrong/ts-node#help-my-types-are-missing) doesn't scan for loose `.d.ts` files; it requires a rigid, explicit config for every `.d.ts` file:

> **TypeScript Node does not use `files`, `include` or `exclude`, by default.** This is because a large majority projects do not use all of the files in a project directory (e.g. Gulpfile.ts, runtime vs tests) and parsing every file for types slows startup time. Instead, `ts-node` starts with the script file (e.g. `ts-node index.ts`) and TypeScript resolves dependencies based on imports and references.
>
> For global definitions, you can use the `typeRoots` compiler option. **This requires that your type definitions be structured as type packages** (not loose TypeScript definition files). More details on how this works can be found in the TypeScript Handbook.

Prior to realizing that `@babel/traverse` is already available in [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/babel__traverse), I wrote my own custom `.d.ts` file for it and tried using `ts-node`. However, even after following [their instructions](https://github.com/TypeStrong/ts-node#help-my-types-are-missing), I kept getting this error:

```
TSError: ⨯ Unable to compile TypeScript:
scripts/lint-mdc.ts:37:40 - error TS7016: Could not find a declaration file for module '@babel/traverse'. '/Users/advorak/dev/mdc-web/node_modules/@babel/traverse/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/babel__traverse` if it exists or add a new declaration (.d.ts) file containing `declare module '@babel/traverse';`

37 import {default as babelTraverse} from '@babel/traverse';
                                          ~~~~~~~~~~~~~~~~~
scripts/lint-mdc.ts:142:23 - error TS7006: Parameter 'nodePath' implicitly has an 'any' type.

142     ExportDeclaration(nodePath) {
                          ~~~~~~~~
```

`ts-node-dev`, on the other hand, worked out of the box with the custom `.d.ts` file, so it seemed like the easier option.

However, now that we don't need a custom `.d.ts` file, we can switch back to `ts-node`.